WPMedia/fusion-news-theme-blocks
# TMEDIA-737: Tag Title block

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description
_Was testing my own knowledge of doing some V1-> V2 migration, so I grabbed the Tag Title Block as tribute. This ticket is actually in *next* sprint, so I appreciate the out of cadence reviews._

## Jira Ticket
- [TMEDIA-737](https://arcpublishing.atlassian.net/browse/TMEDIA-737)

## Acceptance Criteria
- Tag Title block is migrated to the new architecture.

## Test Steps
1. Checkout this branch `git checkout TMEDIA-737/tag-title-block`
2. Run storybook with the following command `npm run storybook`
3. Ensure that the changes made by going to V2 have no impact aside from discrepancies listed here: https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/3138945094/Blocks+Design+Discrepancies

## Effect Of Changes

### Before
The tag title block used news-theme-css to apply styling on the v1 architecture.
https://canary--5f91e4721ccaba0022a10782.chromatic.com/?path=/story/blocks-tag-title--with-name-and-description&globals=backgrounds.grid:false;backgrounds.value:transparent&id=blocks-tag-title--with-name-and-description&viewMode=story
<img width="391" alt="Screen Shot 2022-06-13 at 6 42 38 PM" src="https://user-images.githubusercontent.com/13948125/173459547-45300fe9-11c6-48b9-83ba-baaf7a64c433.png">

### After
The tag title block uses the `stack`, `heading`, and `paragraph` components and is aligned with the new Enhanced Styling architecture.

<img width="386" alt="Screen Shot 2022-06-13 at 6 43 22 PM" src="https://user-images.githubusercontent.com/13948125/173459597-763df348-e950-4666-bd76-69a8f562f1b5.png">


## Dependencies or Side Effects
- Styles have only been added to `news` theme, not `commerce` - so please make sure to only check that in storybook.
- Follow up PR's for *feature pack* and *manifest* need to be made upon approval here.
  - _Do I also need to add to components? I assume no since they care not for blocks_

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
